modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/tests/ProcessBuilderKaleoDesignerReact.testcase

```diff
@@ -231,6 +222,33 @@ definition {
 
 	}
 
+	@description = "LPS-140056 - Verify that the user is able to edit the transition ID"
+	@ignore = "Test Stub"
+	@priority = "4"
+	test CanEditTransitionID {
+		Workflow.openWorkflowListView();
+
+		Workflow.addNewDefinition(workflowDefinitionTitle = "New Workflow Definition");
+
+		ProcessBuilderKaleoDesignerReact.addConnector(
+			connectorIn = "end",
+			connectorLabel = "Start-End Connector",
+			connectorOut = "start",
+			key_direction = "right");
+
+		ProcessBuilderKaleoDesignerReact.editIDConnector(
+			connectorLabel = "START-END CONNECTOR",
+			newConnectorID = "New ID");
+
+		Button.clickPublish();
+
+		Workflow.viewWorkflowDefinition(workflowDefinitionTitle = "New Workflow Definition");
+
+		ProcessBuilderKaleoDesignerReact.selectConnector(connectorLabel = "START-END CONNECTOR");
+
+		ProcessBuilderKaleoDesignerReact.viewConnectorID(ID = "New ID");
```

Comment:
Hi @sirfabios !
This PR is failing SF because of the variable 'ID' in 'CanEditTransitionID' should start with a lowercase letter.
Could you, please, correct it and try the SF again?
Thank you!